src/main/java/org/folio/circulation/resources/CheckInByBarcodeResource.java

```diff
@@ -120,6 +124,7 @@ private void checkin(RoutingContext routingContext) {
         processAdapter::updateLoan, CheckInProcessRecords::withLoan))
       .thenComposeAsync(updateItemResult -> updateItemResult.after(
         patronActionSessionService::saveCheckInSessionRecord))
+      .thenComposeAsync(r -> r.after(overdueFineCalculatorService::calculateOverdueFine))
```

Comment:
Please can all of the permissions needed for all operations that are used in this process be added to the appropriate permission set in the module descriptor

Reply:
Added (in new PR)